cedar-policy-generators/src/expr.rs

```diff
@@ -251,7 +251,24 @@ impl<'a> ExprGenerator<'a> {
                             self.generate_expr(max_depth - 1, u)?,
                             attr_name,
                         ))
-                    })
+                    },
+                    7 => {
+                        let tag_name = uniform!(u,
+                            self.generate_expr(max_depth - 1, u)?,
+                            ast::Expr::val(self.schema.arbitrary_attr(u)?.0.clone())
+                        );
+                        let e = self.generate_expr(max_depth - 1, u)?;
+                        Ok(ast::Expr::get_tag(e, tag_name))
```

Comment:
Noting that you do generate `get_tag` also

Reply:
Good catch.  We do not generate `get_tag` in all the places we would need to -- in particular we do not generate `get_tag` in the type-directed generators in this PR.  I will just remove this from this PR for clarity.